core/src/main/java/org/apache/brooklyn/location/ssh/SshMachineLocation.java

```diff
@@ -1016,12 +1015,8 @@ public String findPassword() {
     /** @deprecated since 1.0.0; mutex-related methods are now accessible via {@link #mutexes()} */
     @Override
     @Deprecated
-    public void acquireMutex(String mutexId, String description) throws RuntimeInterruptedException {
-        try {
-            mutexes().acquireMutex(mutexId, description);
-        } catch (InterruptedException ie) {
-            throw new RuntimeInterruptedException("Interrupted waiting for mutex: " + mutexId, ie);
-        }
+    public void acquireMutex(String mutexId, String description) {
```

Comment:
I would leave the `throws` in place, and change the `MutexSupport.acquireMutex` to also declare `throws RuntimeInterruptedException`.  Because it's runtime you can ignore it as a caller if you want, but the tooling support in IDEs will get some help for anyone who does want to catch it.

Reply:
Effective Java (Item 62) recommends against this for unchecked exceptions, although I've no strong feelings.